Providers/Extras/Scripts/logrotate.conf

```diff
@@ -11,6 +11,6 @@
 /var/opt/omi/log/dscdetailed.log {
     rotate 5
     sharedscripts
-    size 5M
+    size 50M
```

Comment:
How much time it took us to get to 5M based on our testing when consistency/pull running on machine ?

Reply:
I have increase the limit to 50 MB, still waiting for data after filtering some events.

---
In reply to: [140369447](https://github.com/Microsoft/PowerShell-DSC-for-Linux/pull/327#discussion_r140369447) [](ancestors = 140369447)

Reply:
it is ~9MB/day with update management and change tracking enabled.

---
In reply to: [140370840](https://github.com/Microsoft/PowerShell-DSC-for-Linux/pull/327#discussion_r140370840) [](ancestors = 140370840,140369447)